packages/design-system-docs/src/pages/styles/typography/_typography.docs.scss

```diff
@@ -26,6 +26,37 @@ Markup: typography-body.example.html
 Style guide: styles.typography.body
 */
 
+/*
+List
+
+Lists organize written information for users.
+
+Markup: list.example.html
+
+### When to use
+
+- Use an ordered list when you need to display text in some ranking, hierarchy, or series of steps.
+- Use unordered lists to display text in no specific order.
```

Comment:
Curious - do we have guidance on when to use an unstyled list?

Reply:
No we currently do not